packages/components/tests/acceptance/components/hds/alert-test.js

```diff
@@ -0,0 +1,22 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'dummy/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | components/hds/alert', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /components/alert', async function (assert) {
+    await visit('/components/alert');
+
+    assert.strictEqual(currentURL(), '/components/alert');
+  });
+
+  test('Components/alert page passes a11y automated checks', async function (assert) {
```

Comment:
```suggestion
  test('Components/alert page passes automated a11y checks', async function (assert) {
```

Reply:
I went through and added this in a bulk-feature way.